src/bindings/bnd_surface.cpp

```diff
@@ -34,6 +34,31 @@ std::tuple<BND_NurbsSurface*, int> BND_Surface::ToNurbsSurface(double tolerance)
   return std::make_tuple(new BND_NurbsSurface(p_NurbForm, &m_component_ref), accuracy);
 }
 
+BND_TUPLE BND_Surface::FrameAt(double u, double v) {
+  ON_Plane frame;
+  bool success;
+  if (m_surface != nullptr)
+  {
+    success = m_surface->FrameAt(u, v, frame);
+    if (!success) 
+    {
+      frame = nullptr;
```

Comment:
@sbaer curious as to what should be done here with the ON_Plane frame. I've seen examples where some object gets deleted but I got `cannot delete expression of type 'ON_Plane'`, so I removed that.

Reply:
frame is created on the stack which means it is never null. I would rewrite the code to be 

```
BND_TUPLE BND_Surface::FrameAt(double u, double v) {
  ON_Plane frame;
  bool success = false;
  if (m_surface != nullptr)
  {
    success = m_surface->FrameAt(u, v, frame);
  }

  BND_TUPLE rc = CreateTuple(2);
  SetTuple(rc, 0, success);
  SetTuple(rc, 1, BND_Plane::FromOnPlane(frame));
  return rc;
}
```